bridge/src/message_to_mainnet.rs

```diff
@@ -41,10 +41,10 @@ impl MessageToMainnet {
 		let hash = web3_log.transaction_hash
 			.ok_or(Error::from_kind(ErrorKind::Msg("`log` must be mined and contain `transaction_hash`".into())))?;
```

Comment:
this could be also shortened to:

```rust
let hash = web3_log.transaction_hash.ok_or_else(|| "`log` must be mined and contain `transaction_hash`")?;
```

Reply:
done